src/log-analytics-solution/azext_log_analytics_solution/_params.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from ._validators import validate_workspace_resource_id
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('monitor log-analytics solution create') as c:
+        c.argument('resource_group', resource_group_name_type)
```

Comment:
unnecessary to add `resource_group` here as it's a global argument added by cli core automatically.
please also remove it in other methods.